### PR TITLE
PL translations - package #2

### PR DESCRIPTION
**Pull request by ant2753**
Second pack of translations

### CODE REVIEW

File: frontend/messages/pl.json [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+	"helpLayerSoaringLayerDepth2": "to obszar atmosfery, w którym możemy spodziewać się termiki i latania. Głębokość warstwy granicznej mówi nam, jak wysoko możemy wznieść się. Przykładowo wartość 850 m oznacza, że możemy wznieść się do wysokości 850 m nad poziomem gruntu. Do lotów przelotowych preferowane są wartości wyższe niż 750 m.",
+	"helpLayerSoaringLayerDepth3": "W obecności chmur cumulusowych głębokość warstwy granicznej ograniczamy do podstawy chmur. Na przykład, jeśli głębokość warstwy granicznej wynosi 1500 m, ale podstawa chmur cumulusowych znajduje się na wysokości 1200 m nad poziomem gruntu, pokażemy głębokość warstwy granicznej wynoszącą 1200 m.",
+	"helpLayerSoaringLayerDepth4": "Skala kolorów jest pokazana w prawym dolnym rogu ekranu.",
+	"helpLayerThermalVelocity": "Prędkość termiki szacuje się na podstawie wysokości warstwy granicznej i ocieplenia gruntu. Skala kolorów jest pokazana w prawym dolnym rogu ekranu.",
+	"helpMeteogramIntro": "Meteogramy pokazują prognozę pogody dla wybranej lokalizacji w czasie. Oto przykład trzydniowego meteogramu, który stworzyliśmy w celach ilustracyjnych:",
+	"helpMeteogramTopRow1": "-",
```
Comment (julienrf) [inline frontend/messages/pl.json:116]: I noticed you often use this character. Is it correct or temporary?

**Review by julienrf — COMMENTED** [review]
Thank you so much @ant2753, it seems the translation is now complete. Could you please confirm that the way you translated “The” into “-” is correct?

**Comment by ant2753** [comment]
"-" might be where I went wrong. There is no "definite article" in polish corresponding to "the" and there should be "nothing" / "null" where I used "-". I did not know how to differentiate 1) translation not provided 2) correct translation is null/nothing for this context. Please advise how to proceed with that

**Comment by ant2753** [comment]
With regards to this:
"layerSoaringLayerDepth": "Wysokość warstwy granicznej",
I would shorten to:
"Wysokość warstwy gr."

I tried to make this change in Fink. Not sure if succeeded to include in PullRequest.

Dash character "-" in place of "The" looks well in "help" - like non-numbered enumeration which looks like proper formatting for the context.

Both versions look well: with or without dash - please choose yourself (perhaps following consistency to other languages).

Yes, I believe translations are correct, please push to main.

**Comment by ant2753** [comment]
I can see now: ""Wysokość warstwy gr." was committed with today's date

**Comment by julienrf** [comment]
Merged, thank you! Please sync [your fork](https://github.com/ant2753/soaringmeteo/tree/main) before working on the translations again :)

**Comment by julienrf** [comment]
How do we say “Polish” in Polish? Is it “Polski”?

**Comment by ant2753** [comment]
Yes. "Polski" is proper menu entry for language choice

**Pull request merged**